assembly/std/index.ts

```diff
@@ -215,57 +215,86 @@ export function getOpKeys(): Array<StaticArray<u8>> {
 }
 
 /**
- * Get all keys from datastore
+ * Get keys from datastore
+ *
+ * @param prefix - the prefix to filter the keys (optional)
  *
  * @returns - a list of key (e.g. a list of byte array)
  */
-export function getKeys(): Array<StaticArray<u8>> {
-  let keysSer = env.getKeys();
+export function getKeys(
+  prefix: StaticArray<u8> = new StaticArray<u8>(0),
+): Array<StaticArray<u8>> {
+  let keysSer = env.getKeys(prefix);
   return derKeys(keysSer);
 }
 
 /**
  * Get all keys from datastore
  *
  * @param address - the address in the datastore
+ * @param prefix - the prefix to filter the keys (optional)
+ *
  * @returns - a list of key (e.g. a list of byte array)
  */
-export function getKeysOf(address: string): Array<StaticArray<u8>> {
-  let keysSer = env.getKeysOf(address);
+export function getKeysOf(
+  address: string,
+  prefix: StaticArray<u8> = new StaticArray<u8>(0),
+): Array<StaticArray<u8>> {
+  let keysSer = env.getKeysOf(address, prefix);
   return derKeys(keysSer);
 }
 
 /**
- * Internal function - used by getOpKeys
+ * Read the number of keys from Uint8Array little endian format
+ *
+ * @param arr - Uint8Array
+ * @remarks - array length must be equal to 4
```

Comment:
Have you `npm run doc` to see the rendering of a remarks bellow a param?

"array length must be equal to 4" 
does it means:
- it is a fixed-length array
OR
- the length of the array is encoded with 4 bits?

Reply:
Thanks for your comments, you are right, it was not clear. I changed the code and the comment a bit to make it more understandable. I am adding tests for get_keys_for and getKeys in vm.js and vm-mock.spec because they are missing.

Reply:
good job!